src/core.jsx

```diff
@@ -59,26 +59,13 @@ export default (
     return class extends Component {
       static displayName = displayName
 
-      state = {
-        props: {},
-        print: true,
-      }
-
-      omitLoadInProps = (props) => {
-        const isLoadAFunction = isFunction(props[loadFunctionName])
+      constructor(props, context) {
+        super(props, context)
 
-        if (isLoadAFunction) {
-          this.setState({
-            props: {
-              ...props,
-              [loadFunctionName]: undefined,
-            },
-          })
-        } else {
-          this.setState({ props })
+        this.state = {
```

Comment:
Why don't you init state as a class property directly ?

Reply:
You mean like we did before ?
Because there was a babel error, I could set the new transform/plugin from V7, but I feel this is not that hard to write this constructor

Reply:
Sure and we will have a constructor anyway because willMount is deprecated :+1: